modules/editorial-metadata/editorial-metadata.php

```diff
@@ -50,6 +49,12 @@ public static function init(): void {
 
 		// Load block editor CSS
 		add_action( 'enqueue_block_editor_assets', [ __CLASS__, 'load_styles_for_block_editor' ] );
+
+		// Add metadata fields to term
+		add_filter( 'get_term', [ __CLASS__, 'add_metadata_to_term' ], 10, 1 );
```

Comment:
Both of these allow us to massage the terms to add metadata info in, and save us an extra call when doing gets.